schema.sql

```diff
@@ -1,5 +1,43 @@
 /* Database schema to keep the structure of entire database. */
 
+PSQL
+CREATE DATABASE vet_clinic /* Creating vet clinic database */
+\c vet_clinc /* Connect to the vet_clinic db using psql*/
 CREATE TABLE animals (
-    name varchar(100)
-);
\ No newline at end of file
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100),
+    date_of_birth DATE,
+    escape_attempts INTEGER,
+    neutered BOOLEAN,
+    weight_kg DECIMAL(10, 2)
+);
+/* Create column called species */
+ALTER TABLE animals
+ADD species VARCHAR(255);
+
+
+psql
+/c mydatabases
+CREATE TABLE owner (
+  id SERIAL PRIMARY KEY,
+  full_name VARCHAR(255),
+  age INTEGER
+);
```

Comment:
- [ ] Kindly update your queries. As mentioned by the previous reviewer you are creating `Owner` tables while it should be `Owners`. Let's make sure we remove this typo. Thank you.